src/components/PopoverContainer/PopoverContainer.module.css

```diff
@@ -10,4 +12,8 @@
   overflow: auto;
   box-shadow: var(--eds-box-shadow-md);
   background-color: var(--eds-theme-color-background-neutral-default);
+
+  &:focus-visible {
+    @mixin focus;
+  }
```

Comment:
We actually [need to have a fallback](https://github.com/chanzuckerberg/edu-design-system/blob/main/docs/CODE_GUIDELINES.md#focus-states) for `focus-visible` use since we do support some browser versions that don't support `focus-visible`

Reply:
I may need to revisit how this [polyfill](https://www.npmjs.com/package/focus-visible) works , and what it means for things we style in EDS and consuming apps. I'll look at other examples to see what we should do.

What's the browser we support that doesn't do this? https://caniuse.com/mdn-css_selectors_focus-visible

Reply:
Ah completely forgot we had the polyfill for `focus-visible`, think it should work then

Reply:
Looking at the list of [supported browsers](https://czi.atlassian.net/wiki/spaces/LP/pages/2057733851/Edu+Tech+Inclusive+Browser+Support+Policy) I think it's Edge 84-85 and Safari 14-15.3, and hopefully firefox gets autoprefixed for it

Reply:
The polyfill is in `traject` and `along`, not EDS, so we have been adding the fallback for `:focus-visible`. I think at the time we decided to use `:focus-visible` + the `@supports` fallback, the polyfill wasn't in `traject` yet. I would make the case that we should keep the fallback because we don't know if the app using EDS has the polyfill, but if you think we should drop the fallback because the polyfill is now in place in our 2 current apps, we should update the docs and code in EDS.

https://github.com/chanzuckerberg/edu-design-system/blob/e0288ce88d4cccbf5a1e3fc5cd79b5b8ee0ce4fd/docs/CODE_GUIDELINES.md?plain=1#L121

Reply:
@dierat good note. I will make a ticket for fixing this. as those new pilot get set up, there'd be an extra step for each of them to remember to add that polyfill, set it up properly, etc. yuck!